### ISSUE TITLE
Duplicated Nested Close Buttons after update

### ISSUE THREAD

**Issue by pantomash** [Stabzs/Angular2-Toaster#235]
I have overrided close button html. In version 8.0.0 generated code was looking fine. 
After update to newest version 11.0.1 that was needed to update Angular, generated code looks different. 
Now my custom close button is wrapped by generated button element.

```
 toasterConfig = new ToasterConfig({
    tapToDismiss: true,
    mouseoverTimerStop: true,
    animation: 'fade',
    timeout: {
      error: NOTIFICATION_TIMEOUT_NONE,
      success: NOTIFICATION_TIMEOUT,
      warning: NOTIFICATION_TIMEOUT,
    },
    showCloseButton: true,
    closeHtml: `<button aria-label="Close" class="toast-close-button">x</button>`,
  })
```

![image](https://user-images.githubusercontent.com/45421850/123162188-99ff4080-d470-11eb-8eee-65412ba92c8e.png)

**Comment by skrzepij** [comment]
I think it will be great to set a new optional `@Input`  for the aria-label value as this is a missing part of accessibility in the hardcoded `button` element.

**Comment by Stabzs** [comment]
Correct, in 11.0.0 styles and layout were reworked to create more consistent layout across devices.  It also cuts down on the amount of boilerplate magic string content that needs to be provided just to add a button.

It looks like some of the documentation incorrectly still suggests adding explicit `<button></button>` html for the close button which needs to be updated.  In your case @pantomash, your closeHtml simply becomes closeHtml: `x`.

However, this doesn't solve the issue of binding aria-label and/or classes.  I think your idea of an input binding @skrzepij should work and I'll look into getting that change in soon.
